src/utils/constants.ts

```diff
@@ -157,6 +157,11 @@ enum RegionMapping {
   'north-america' = 5,
 }
 
+enum WalletMode {
+  noUI = 0,
```

Comment:
```suggestion
  NoUI = 0,
```

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/91/commits/c125b895d32a77d47d421bb30bb6673ec45c8196 and https://github.com/arcana-network/developer-dashboard/pull/91/commits/7610f209ae34c6df76117d38ae326ec6dc9f57bf